content/docs/editing/blocks.md

```diff
@@ -125,6 +125,43 @@ We have defined the structure of our 3 blocks (`content`, `features`, `hero`), a
 
 > Note, since our contentBlock uses the [markdown plugin](/docs/reference/toolkit/fields/markdown/) which is not registered out of the box, we will need to manually import the `react-tinacms-editor` plugin.
```

Comment:
I know its not part of this PR but you could probably link to this section (guides/tina-cloud/add-tinacms-to-existing-site/markdown-plugin/) of the guide to provide more context of how to add the `react-tinacms-editor` plugin.